posts/tests.py

```diff
@@ -1,3 +1,91 @@
-# from django.test import TestCase
+import pytest
+from .models import Post
+from .models import Resume
+from django.contrib.auth.models import User
 
-# Create your tests here.
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "testpass"
+EMAIL = "testuser@gmail.com"
+DESCRIPTION = "this is a test resume"
+FILE1 = "Alon_Shakaroffs_resume.pdf"
+
+
+# -----------------------------------------------fixtures----------------------------------------------------------
+
+@pytest.fixture
+def new_user():
+    return User(username=USERNAME, first_name=FIRSTNAME, last_name=LASTNAME, password=PASSWORD, email=EMAIL)
+
+
+@pytest.fixture
+def persist_user(new_user):
+    new_user.save()
+    return new_user
+
+
+@pytest.fixture
+def new_resume(new_user):
+    return Resume(author=new_user, description=DESCRIPTION)
+
+
+@pytest.fixture
+def persist_resume(new_resume):
+    new_resume.author.save()
+    new_resume.save()
+    return new_resume
+
+
+# -----------------------------------------------Resume tests------------------------------------------------------
+
+@pytest.mark.django_db()
+class TestResume:
+    # Check that the Resume values are the same as the Resume inputs.
+    def test_new_resume(self, new_resume):
+        assert new_resume.author.username == USERNAME
+        assert new_resume.description == DESCRIPTION
+
+    # Check if the Resume can be referred as a post.
+    def test_resume_as_post(self, new_resume):
+        assert len(Post.objects.all()) == 0
+        new_resume.author.save()
+        new_resume.save()
+        assert isinstance(new_resume, Post)
+        assert len(Post.objects.all()) == 1
+        post = Post.objects.all().first()
+        assert hasattr(post, 'resume') is True
+        assert hasattr(post, 'poll') is False
```

Comment:
```suggestion
        assert hasattr(post, 'resume')
        assert not hasattr(post, 'poll')
```
It is redundant to check if True or False

Reply:
Ok, thank you, I changed that 👍 @abiton1